lib/v2/caixin/latest.js

```diff
@@ -1,18 +1,16 @@
 const got = require('@/utils/got');
 const cheerio = require('cheerio');
-const { parseDate } = require('@/utils/parse-date');
-const timezone = require('@/utils/timezone');
 const { art } = require('@/utils/render');
 const path = require('path');
 
 module.exports = async (ctx) => {
-    const li_r = await got('https://www.caixin.com/search/scroll/0.jsp');
+    const data = await got('https://gateway.caixin.com/api/dataplatform/scroll/index');
 
-    const $ = cheerio.load(li_r.data);
-    const list = $('div.news_content dl dd')
-        .toArray()
-        .map((dd) => ({
-            link: $(dd).find('a').first().attr('href').replace('http://', 'https://'),
+    const list = JSON.parse(data.body)
```

Comment:
`data.data` is already a parsed object. No need to parse it manually again.

Reply:
You mean…Just write `data.data.articlelist`?